lib/tasks/payment_breakdown.rake

```diff
@@ -54,15 +63,20 @@ namespace :payment_calculation do
     table.add_row ["Banding", "Output price  (PP)", "Output price  (monthly)"]
     table.add_separator
     output_payments.each { |row| table.add_row(row) }
+    table.add_separator
+    table.add_row ["", "Per participant", "Total Uplift"]
+    table.add_separator
+    table.add_row(["Uplift", uplift_payment[:per_participant], uplift_payment[:sub_total]])
 
     output = <<~RESULT
       Based on #{number_to_delimited(total_participants.to_i)} participants and #{per_participant_in_bands}
     RESULT
 
     logger.info table
     logger.info output
-  rescue StandardError
-    logger.info "Lead provider for '#{ARGV[1]}' not found"
+  rescue StandardError => e
+    puts e.message
+    puts e.backtrace
```

Comment:
If we're just spitting out the message and trace do we need to `rescue` at all?

Reply:
It was spitting out the wrong message before. Telling that the lead provider could not be found.

Reply:
I can see that was the case in the removed code, but do we need the replacement code, or can we just delete the rescue entirely?